docs/highlights/2020-09-18-adaptive-concurrency.md

```diff
@@ -0,0 +1,67 @@
+---
+last_modified_on: "2020-09-18"
+$schema: ".schema.json"
+title: "Adaptive concurrency. Optimizing performance & reliability."
+description: "How Vector optimizes performance and increases reliability of your entire observability infrastructure."
+author_github: "https://github.com/binarylogic"
+pr_numbers: [3094]
+release: "0.11.0"
+hide_on_release_notes: false
+tags: ["type: announcement", "domain: networking"]
+---
+
+Vector 0.11 includes a new adaptive conccurrency feature designed to
+automatically optimize network bandwidth, backoff when it senses trouble,
+and make your observability infrastructure significantly more performant and
+reliable.
+
+## Adaptive Concurrency. The Elevator Pitch.
+
+We cover this feature in-depth in our [announcement blog post][announcement],
+but here's the skinny.
+
+One of the most difficult things to do in distributed systems is coordinating
+communication in a way that optimizes performance without risking reliability.
+This is often achieved through rate-limits. In theory, rate-limits sound simple,
```

Comment:
s/rate-limits/rate limits throughout

Also, I'd add a qualifier to rate limits. Maybe "constant rate limits" or "static rate limits." Something to emphasize the brute force nature of that approach.